pwiz_tools/Skyline/Model/Results/ScanProvider.cs

```diff
@@ -40,6 +40,12 @@ public class TransitionFullScanInfo
         public double? ExtractionWidth;
         public IonMobilityFilter _ionMobilityInfo;
         public Identity Id;  // ID of the associated TransitionDocNode
+
+        public bool MatchMz(double mz)

```

Comment:
The original logic in GraphFullSpectrum was this:
!(mz <= transition.ProductMz.Value - transition.ExtractionWidth/2 || mz > transition.ProductMz.Value + transition.ExtractionWidth/2))
which is the same I currently have in the MatchMz() method. If you think this is a bug I'll fix it.

Reply:
The two should match, as Nick points out. This is a very slight difference <= and < are virtually identical with double values. I would have thought the filter would be "mz >= ProductMz.Value - ExtractionWidth / 2 &&
mz <= ProductMz.Value + ExtractionWidth / 2", but I can see that Nick is right and I must have made it left inclusive and right exclusive. Better add a comment here explaining that the logic was to match SpectrumFilterPair.FilterSpectrumList so that this discussion is not forever lost.